core-java-modules/core-java-date-operations-2/README.md

```diff
@@ -7,4 +7,5 @@ This module contains articles about date operations in Java.
 - [Checking If Two Java Dates Are on the Same Day](https://www.baeldung.com/java-check-two-dates-on-same-day)
 - [Converting Java Date to OffsetDateTime](https://www.baeldung.com/java-convert-date-to-offsetdatetime)
 - [How to Set the JVM Time Zone](https://www.baeldung.com/java-jvm-time-zone)
+- [Extracting the Day of the Week](http://inprogress.baeldung.com/wp-admin/post.php?post=184664&action=edit&classic-editor)
```

Comment:
Let's remove it for now - `inprogress` is not exposed publically and these links get added after an article gets published

Reply:
I updated the `README.md': [commit](https://github.com/eugenp/tutorials/pull/8939/commits/c6811a2983fe567b001ba8261e53fff10dc0c94d)